src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -74,12 +77,12 @@ class Device(object):
         'vendor',
     ]
 
-    def __init__(self, path):
+    def __init__(self, path, lvs=None):
         self.path = path
+        self.lvs = lvs or []
```

Comment:
@jan--f The CI failures were probably due to this line. If empty list gets assigned, `self.lvs.get()` won't work.